src/Surfnet/StepupMiddleware/ApiBundle/Controller/RecoveryTokenController.php

```diff
@@ -65,7 +65,7 @@ public function collection(Request $request): JsonCollectionResponse
             sprintf('Received search request for recovery tokens with params: %s', $request->getQueryString()),
         );
         $query = new RecoveryTokenQuery();
-        $query->identityId = new IdentityId($request->get('identityId'));
+        $query->identityId = $request->get('identityId') ? new IdentityId($request->get('identityId')) : null;
```

Comment:
Am I correct that I see you removed the param converter for the IdentityId? I'm OK with that, but I want to be sure.

Reply:
Nope, but if you inject null into this VO this would otherwise fail